src/index.js

```diff
@@ -1,10 +1,8 @@
 const server = require('./server');
 
-const host = process.env.HOST || 'blah';
-const port = process.env.PORT || 1;
+const host = process.env.HOST || '127.0.0.1';
+const port = process.env.PORT || 8080;
```

Comment:
Remember, default values should try to be "smoke test" valid because if there's no env, that's what's getting used.

Reply:
Good catch! I think i put these values here to make sure that the environment variables were working but never changed them back